### PR TITLE
Tooling Cleanup (Docker & Script Rewrites)

### PR DESCRIPTION
**Pull request by azenla**
Rewrite scripts to practice DRY and utilize Docker for simplifying the environmental setup.

### CODE REVIEW

File: tools/docker_run.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```
Comment (charbull) [inline tools/docker_run.sh:1]: can you please update the readme on how to you the docker part if it is ready?

Comment (azenla) [reply tools/docker_run.sh:1]: Yes, I will write some docs on this today

Comment (trav3711) [reply tools/docker_run.sh:1]: + 1

**Comment by tasodorff** [comment]
@azenla please make recommended readme changes

**Review by trav3711 — COMMENTED** [review]
Waiting for requested doc updates

**Comment by azenla** [comment]
I have some docs in progress, I am figuring out what to do about the credentials now, as the way I am mounting them is probably not ideal.

File: tools/abel/get_token.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+create_token_request() {
+  TOKEN_REQUEST_PATH="$(mktemp)"
+  cat > "${TOKEN_REQUEST_PATH}" <<EOF
+{"scope":["https://www.googleapis.com/auth/spreadsheets"],"lifetime":"3600s"}
+EOF
+  echo "${TOKEN_REQUEST_PATH}"
+}
+
+acquire_access_token() {
```
Comment (charbull) [inline tools/abel/get_token.sh:26]: please sync with @trav3711 this is deprecated, he can show you the new path

File: tools/scripts/docker_entry.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -e
+
+source /env/bin/activate
```
Comment (charbull) [inline tools/scripts/docker_entry.sh:4]: nice !

**Review by charbull — COMMENTED** [review]
ping when you are ready

**Review by trav3711 — APPROVED** [review]

**Pull request merged**